make.py

```diff
@@ -0,0 +1,249 @@
+"""
+Helper script to build langkit and its dependencies.
+"""
+
+from __future__ import absolute_import, division, print_function
+
+from contextlib import contextmanager
```

Comment:
The fact that the stylechecker does not complain about this line makes me realize that the stylechecker does not run on this script! Please move it to the `scripts` directory.